syntax="proto3";

message CreateProjectRequest{
    string ProjectId = 1;
    string ProjectName = 2;
}
message CreateProjectResponse{}

message StatRequest{}

message StatResponse{
    string version = 1;
}


service Catalog {
    rpc CreateProject(CreateProjectRequest) returns (CreateProjectResponse){}
    rpc Stat(StatRequest) returns (StatResponse){}

}

message CreateDataset {
    string dataset_id = 1;
    string project_id = 2;
}






